src/Symfony/Bridge/Monolog/Command/ServerLogCommand.php

```diff
@@ -34,6 +34,7 @@ class ServerLogCommand extends Command
     private $handler;
 
     protected static $defaultName = 'server:log';
+    protected static $defaultDescription = 'Starts a log server that displays logs in real time';
 
     public function isEnabled()
```

Comment:
logic of `isEnabled` should be duplicated in FrameworkExtension (to remove the tag) otherwise this command will always be enabled (same for all Command with `isEnabled` method)

Reply:
good catch, `ServerLogCommand` is the only one, updated.